lib/index.js

```diff
@@ -1,51 +1,12 @@
 const $ = require('jquery')
 const Ship = require('../lib/ship');
+// const Asteroid = require('../lib/asteroid');
```

Comment:
Was this commented out because it is required when the 'asteroids' file is required?